llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -106,6 +106,27 @@ bool PostPipeliner::canAccept(MachineBasicBlock &LoopBlock) {
   return true;
 }
 
+int PostPipeliner::getResMII(MachineBasicBlock &LoopBlock) {
+  // For each instruction, find the first cycle in which it fits and collect the
+  // maximum
+  std::vector<uint64_t> Scoreboard(NInstr, 0);
+  int MII = 1;
+  for (auto &MI : LoopBlock) {
+    auto *SlotInfo = TII->getSlotInfo(TII->getSlotKind(MI.getOpcode()));
```

Comment:
Might want to start using `SelectedAltDescs` to prepare for the future?

Reply:
Since this is just a lower bound, I think I want an optimistic value, not values established in a different schedule. And I think the future is to run the pipeliner first. I guess not counting unknown slots at all would be the proper thing to do.